projects/angular/src/utils/_theme.dark.clarity.scss

```diff
@@ -22,7 +22,6 @@
 // - Signpost
 // - Spinner
 // - Stack View
-// - Table
```

Comment:
Maybe this was accidentally deleted?

Reply:
It was repeated 2 times removed one of the occurrences.